### PR TITLE
feat: (sfui2-676) breadcrumbs block

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes[ #676](https://vsf.atlassian.net/browse/SFUI2-676)

# Scope of work

Added Bredcrums blocks.

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
All comments are on the old PR https://github.com/vuestorefront/sfui2/pull/706 so please refer to them.

File: packages/sfui/frameworks/vue/components/VsfLink/VsfLink.vue
Comment (dkacper) [inline packages/sfui/frameworks/vue/components/VsfLink/VsfLink.vue:1]: This Vsf component also should not be here. 😉

**Review by dkacper — COMMENTED** [review]
All of my previous comments has been addressed. I like the changes now. However, I can feel we could improve it a bit more. And also, correct me if I'm wrong, but I think VsfLink components should not be here.

File: apps/preview/next/pages/showcases/Breadcrumbs/BreadcrumbsWithIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+              ))}
+            </div>
+          </SfDropdown>
+        </li>
+        {breadcrumbs.map((item) => (
+          <li
+            className="peer hidden sm:flex peer-[:nth-of-type(even)]:before:content-['/'] peer-[:nth-of-type(even)]:before:px-2 peer-[:nth-of-type(even)]:before:leading-5 last-of-type:flex last-of-type:before:font-normal last-of-type:before:text-neutral-500 last-of-type:text-neutral-900 last-of-type:font-medium"
+            key={item.name}
+          >
+            {breadcrumbs.indexOf(item) === 0 ? (
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply apps/preview/next/pages/showcases/Breadcrumbs/BreadcrumbsWithIcon.tsx]: Obviously... thx

File: apps/preview/nuxt/pages/showcases/Breadcrumbs/BreadcrumbsWithIcon.vue [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          </div>
+        </SfDropdown>
+      </li>
+      <li
+        v-for="item in breadcrumbs"
+        :key="item.name"
+        class="peer hidden sm:flex peer-[:nth-of-type(even)]:before:content-['/'] peer-[:nth-of-type(even)]:before:px-2 peer-[:nth-of-type(even)]:before:leading-5 last-of-type:flex last-of-type:before:font-normal last-of-type:before:text-neutral-500 text-neutral-500 last-of-type:text-neutral-900 last-of-type:font-medium"
+      >
+        <SfLink
+          v-if="breadcrumbs.indexOf(item) === 0"
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply apps/preview/nuxt/pages/showcases/Breadcrumbs/BreadcrumbsWithIcon.vue]: done

File: packages/sfui/frameworks/react/components/VsfLink/VsfLink.tsx
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/sfui/frameworks/react/components/VsfLink/VsfLink.tsx:1]: You are right, probably left it when moving from old repo.

File: packages/sfui/frameworks/vue/components/VsfLink/VsfLink.vue
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/VsfLink/VsfLink.vue:1]: removed

**Comment by FRSgit** [comment]
@AdamPawlinski rebase with `v2-develop` branch to get the tests passing

**Pull request merged**